docs/image.md

```diff
@@ -9,7 +9,9 @@ The following defines the relevant fields of the `image` resource spec in more d
 - `builder`: Configuration of the `builder` resource the image builds will use. See more info [Builder Configuration](builders.md).
 - `serviceAccount`: The Service Account name that will be used for credential lookup.
 - `source`: The source code that will be monitored/built into images. See the [Source Configuration](#source-config) section below.
-- `cacheSize`: The size of the Volume Claim that will be used by the build cache.
+- `cache`: Caching configuration, two variants are available:
+  - `volume.request`: Creates a Volume Claim of the given size
```

Comment:
should this be `volume.size` ? `request` seems less user-friendly. anyone else have thoughts?

Reply:
Hmm. I like how request matches the k8s terminology for PVCs.

Reply:
Should this actually be requests and be a RequestsList?

Reply:
```
- `cache`: Caching configuration, two variants are available:
  - `volume.size`: Creates a Volume Claim of the given size
  - `registry.imageTag`: Creates an image with cached contents
 ```